src/main.rs

```diff
@@ -20,5 +30,90 @@ impl edg::ExtendedDependencyGraph<i32> for EmptyGraph {
 }
 
 fn main() {
-    edg::distributed_certain_zero(EmptyGraph {}, 0, num_cpus::get() as u64);
+    let args = parse();
+
+    // Load config for logging to stdout and logfile.
+    if let Ok(_handle) = log4rs::init_config(get_log4rs_config(
+        args.value_of("log-path").unwrap(),
+        match args.value_of("log-level").unwrap().to_lowercase().as_str() {
+            "error" => LevelFilter::Error,
+            "warn" => LevelFilter::Warn,
+            "info" => LevelFilter::Info,
+            "debug" => LevelFilter::Debug,
+            "trace" => LevelFilter::Trace,
+            "off" => LevelFilter::Off,
+            _ => LevelFilter::Off,
+        },
+    )) {
+        info!("Model checking on proposition: {}", args.value_of("proposition").unwrap());
+
+        edg::distributed_certain_zero(EmptyGraph {}, 0, num_cpus::get() as u64);
+    }
+}
+
+/// Define and parse command line arguments
+fn parse() -> ArgMatches<'static> {
+    App::new("OnTheFlyATL")
+        .version("0.1.0")
+        .author("d702e20 <d702e20@cs.aau.dk>")
+        .arg(Arg::with_name("proposition")
+            .short("p")
+            .long("prop")
+            .env("PROPOSITION")
+            .help("The proposition to check for"))
+        .arg(Arg::with_name("input_file")
+            .short("i")
+            .long("input")
+            .env("INPUT_FILE")
+            .help("The input file to generate model from"))
+        .arg(Arg::with_name("json_model")
+            .short("j")
+            .long("json")
+            .env("INPUT_JSON")
+            .help("The json to generate model from"))
+        .arg(Arg::with_name("json_formula")
+            .short("r")
+            .long("jsonformula")
+            .env("JSON_FORMULA")
+            .help("The json to generate formula from"))
+        .arg(Arg::with_name("log-level")
+            .short("o")
+            .long("log-level")
+            .env("LOG_LEVEL")
+            .default_value("info")
```

Comment:
the UNIX philosophy is to not print anything unless there is an error or the user requested it

```suggestion
            .default_value("warn")
```

Reply:
updut

Reply:
Agreed, this was default in d502e19 project. Fixed in d69b7c2